### ISSUE TITLE
Full list item tooltip

### ISSUE THREAD

**Issue by iqnite** [ScratchAddons/ScratchAddons#6992]
### Why this would be helpful

Currently, when a list item overflows the list's width, the only way to fully read it is to copy and paste the item into a text field.

### How this addon works

When the user hovers over a list item, the addon fully shows it in a tooltip.

### Possible alternatives

There have been 2 alternatives:
- Wrapped list items (failed due to Scratch bug)
- Scrollable list items (in ScratchTools; added annoying scrollbars)

### Additional context

<img width="237" alt="image" src="https://github.com/ScratchAddons/ScratchAddons/assets/50552317/0ad01c6c-7188-4e05-b484-a39f1bc1ee3a">

**Comment by DNin01** [comment]
A workaround is to use the Variable manager addon, I think? However, that only works if you're in the editor.

**Comment by mybearworld** [comment]
A simple title text would probably work for this.

**Comment by iqnite** [comment]
> A simple title text would probably work for this (instead of a custom colored tooltip).

We can just use a normal browser tooltip, that image's just a mockup.

**Comment by BroJac5246** [comment]
Might I also propose making this enabled by default? It's unobtrusive and universally helpful.

**Comment by Samq64** [comment]
I think this should be part of `last-edit-tooltip`.

**Comment by mxmou** [comment]
> I think this should be part of `last-edit-tooltip`.

Merging editor and website addons is problematic because of the way the settings page is organized.

**Comment by DNin01** [comment]
> > I think this should be part of `last-edit-tooltip`.
> 
> Merging editor and website addons is problematic because of the way the settings page is organized.

Yeah, I found that out when attempting something and reported it in #6409.

**Comment by Secret-chest** [comment]
> Merging editor and website addons is problematic because of the way the settings page is organized.

Not only thay, but they add tooltips to completely different things.

**Comment by Secret-chest** [comment]
As an alternative, have two scrolling arrows that scroll left or right when hovered.

**Comment by hueychen27** [comment]
For the alternative suggestion, adding scrollbars, you could try removing scrollbars. See: https://codepen.io/Horsey_Guy/pen/OJqPmGR

**Comment by mybearworld** [comment]
> For the alternative suggestion, adding scrollbars, you could try removing scrollbars. See: https://codepen.io/Horsey_Guy/pen/OJqPmGR

I don't think that'd help too much.
